opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/view.py

```diff
@@ -96,6 +96,7 @@ def __init__(
         attribute_keys: Optional[Set[str]] = None,
         aggregation: Optional[Aggregation] = None,
         instrument_unit: Optional[str] = None,
+        aggregation_cardinality_limit: Optional[int] = None,
```

Comment:
Looks like these args are individually documented in the docstring above. Should we add one for this arg?